### PR TITLE
861 improve frontend tests

### PR DESCRIPTION
**Pull request by robrobert99**

### CODE REVIEW

File: .github/workflows/cypress_tests.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: End-to-end tests
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - master
+      - develop
```
Comment (bonndan) [inline .github/workflows/cypress_tests.yml:9]: add `- 245_improve_frontend_tests` here to run against this branch

File: .github/workflows/cypress_tests.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v1
+        with:
+          build: npm run build
+          start: npm start
+          wait-on: http://localhost:3000
```
Comment (bonndan) [inline .github/workflows/cypress_tests.yml:23]: how will that work?

**Review by bonndan — CHANGES_REQUESTED** [review]
Please add some lines in development.md how to run the cypress tests and add a link to the dashboard.

Bonus: Is there a cypress.io badge for the main readme?

**Review by bonndan — APPROVED** [review]

File: development.md [hunk 3]
```diff
@@ -68,13 +68,29 @@ The `LandscapeRespository` gives access to the stored landscapes.
...
-by an observer mechanism on files. These are the files located under the path provided through the `SEED` environment variable.
+The `Indexer` can be triggered either directly through the `ApiController` to index or reindex a landscape, or it is
+triggered by an observer mechanism on files. These are the files located under the path provided through the `SEED`
+environment variable.
 
+**Testing**
+
+Smoke tests:
+
+In Nivio smoke tests are executed, for new commits to develop branch and master branch.
```
Comment (jenarp) [inline development.md:87]: In Nivio, smoke tests are executed for new commits to the develop and master branch.

**Review by jenarp — APPROVED** [review]

**Pull request merged**